MobileWallet/Backup/ICloudBackup.swift

```diff
@@ -161,17 +252,143 @@ class ICloudBackup: NSObject {
     }
 }
 
+// MARK: Reachability
+extension ICloudBackup {
+    private func startObserveReachabilityChanges() throws {
+        reachability = try Reachability()
+        NotificationCenter.default.addObserver(self, selector: #selector(reachabilityChanged(note:)), name: .reachabilityChanged, object: reachability)
+        try reachability?.startNotifier()
+    }
+
+    @objc private func reachabilityChanged(note: Notification) {
+        synchWithiCloud()
+    }
+    private func stopReachability() {
```

Comment:
When you're writing functions that do opposite things try to follow the same naming pattern => `stopObserveReachability`.